404.md

```diff
@@ -1,10 +0,0 @@
----
```

Comment:
Why are you removing this page? I think we still need it.

Reply:
WTF lol. I don't know what happened